server/src/pages/shot/text-tool.js

```diff
@@ -98,23 +101,46 @@ exports.TextTool = class TextTool extends React.Component {
     };
 
     return [
-      <div draggable="true" key="drag" onDragStart={this.onDragStart.bind(this)} style={dragDivStyles}>
+      <div key="drag" style={dragDivStyles}
+        onMouseDown={this.onDragMouseDown.bind(this)} onMouseUp={this.onMouseUp.bind(this)} onMove={this.onMouseMove.bind(this)}>
         <div id="text-input" ref={this.textInput} contentEditable="true" key="text" onKeyDown={this.onKeyDown.bind(this)}
-          onKeyUp={this.onKeyUp.bind(this)} className={`${this.state.textSize} ${this.state.colorName} text`} onFocus={this.moveCursorToEnd.bind(this)}>
```

Comment:
Removing moveCursorToEnd brings back issue of type some text and then change the font color, the cursor is moved to the beginning of the input.